ray-on-gke/platform/modules/gke_standard/main.tf

```diff
@@ -87,3 +96,22 @@ resource "google_container_node_pool" "gpu_pool" {
     }
   }
 }
+
+resource "google_container_node_pool" "multi_host_tpu_pool" {
```

Comment:
The topology is a 2x2x1 which is actually a single host.

Reply:
modified